FirebaseMessaging/Apps/SwiftUISample/SwiftUISample/ContentView.swift

```diff
@@ -0,0 +1,28 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import SwiftUI
+
+struct ContentView: View {
+  var body: some View {
+    Text("Hello, world!")
```

Comment:
Is there any UI we can show that shows the state of the APNs token and/or FCM token?

Reply:
I plan to merge the content view with previous version of test app, but uses AppDelegate so there's some refactor needs to be done. I will do it in another PR.